src/main/resources/biz_de.properties

```
@@ -285,6 +285,8 @@ FileImportJob.requiredFileNotFound = Die Datei '${filename}' wurde im Archiv nic
 FileImportJobFactory.auxFileFlattenDirectoriesParameter = Ordnerstruktur der zusätzlichen Dateien ignorieren
 FileImportJobFactory.auxFileFlattenDirectoriesParameter.help = Gibt an, ob die Ordnerstruktur im Archiv ignoriert werden soll. Wenn gesetzt, werden alle zusätzliche Dateien in das Zielverzeichnis abgelegt ohne Unterordner anzulegen.
 FileImportJobFactory.auxFileMode = Zusätzliche Dateien
+FileImportJobFactory.auxFileParentDirectory = Basispfad
+FileImportJobFactory.auxFileParentDirectory.help = Definiert einen Basispfad (übergeordnetes Verzeichnis), der im Zielverzeichnis verwendet oder erstellt wird, um die zusätzliche Dateien hochzuladen. Verwenden Sie '/', um mehrere Verzeichnisse anzugeben.
```

Not sure about "mehrere Verzeichnisse" ... sounds like multiple independent directories existing "side by side", while a hierarchy of nested directories is meant

---

Suggestions are welcome! 😇

---

What about something like the following? Depends a little on the audience, of course.
```suggestion
FileImportJobFactory.auxFileParentDirectory.help = Definiert einen Ordner innerhalb des Zielverzeichnisses, in den die neuen Dateien hochgeladen werden. Verwenden Sie '/' für einen Pfad mit Unterverzeichnissen. Der Ordner wird nötigenfalls neu angelegt.
```